eslint.config.mjs

```diff
@@ -31,6 +31,7 @@ export default [
       'simple-import-sort': simpleImportSort,
     },
     rules: {
+      'no-console': 'error',
```

Comment:
I added this eslint rule to prevent accidental uses of `console` when the `log` helper functions should be used instead. Accidentally using `console.log` can also be problematic for stdio transport if the message is not a proper JSON RPC message.